README.md

```diff
@@ -1,55 +1,127 @@
 
 ## Setup development environment
 
 To setup your local env, you will need to install Python 3.6+ and Google SDK in a UNIX based operating system.
 
-* ### Clone the repository 
+- ### Clone the repository
 
-    ```
-    $ git clone https://github.com/neu-self-image-experiments/sie-backend.git
-    ```
+  ```
+  $ git clone https://github.com/neu-self-image-experiments/sie-backend.git
+  ```
 
-* ### Installing dependencies
-    If you do not have `venv` package you can first install it:
-    ```
-    $ sudo apt install python3-venv
-    ```
-    Then to create new virtual environment run the following command in your project directory:
-    ```
-    $ sudo python3 -m venv venv
-    ```    
-    You need to activate the virtual environment in order to use it. So activate the virtual environment using the following command:
-    ```
-    $ source venv/bin/activate
-    ```
-    Once activated, you would see the shell prompt will change and it will show the name of the virtual environment that you are currently using. In our case it is venv.
+- ### Installing dependencies
 
-    Now install all the dependencies using:
-    ```
-    (venv) $ sudo pip3 install -r requirements.txt
-    ```
+  If you do not have `venv` package you can first install it:
 
-    Now run the flask server:
-    ```
-    (venv) $ python3 main.py
-    ```
+  ```
+  $ sudo apt install python3-venv
+  ```
+
+  Then to create new virtual environment run the following command in your project directory:
+
+  ```
+  $ sudo python3 -m venv venv
+  ```
+
+  You need to activate the virtual environment in order to use it. So activate the virtual environment using the following command:
+
+  ```
+  $ source venv/bin/activate
+  ```
 
+  Once activated, you would see the shell prompt will change and it will show the name of the virtual environment that you are currently using. In our case it is venv.
+
+  Now install all the dependencies using:
+
+  ```
+  (venv) $ sudo pip3 install -r requirements.txt
+  ```
+
+  Now run the flask server:
+
+  ```
+  (venv) $ python3 main.py
+  ```
 
 * ### Installing Google SDK
-    To install Google Cloud SDK for your OS: https://cloud.google.com/sdk/docs/install#installation_instructions
+
+  To install Google Cloud SDK for your OS: https://cloud.google.com/sdk/docs/install#installation_instructions
 
 * ### Deploying function to Google Cloud
-    Using the gcloud command-line tool, deploy your function from the directory containing your function code with the gcloud functions deploy command:
+
+  Using the gcloud command-line tool, deploy your function from the directory containing your function code with the gcloud functions deploy command:
+
+  ```
+  gcloud functions deploy FUNCTION_NAME --runtime python37 --trigger-http --allow-unauthenticated
```

Comment:
nit: should we do `python37` or `python38`? I think I deployed the function with `python38`

Reply:
Python 37 worked for me. The google docs example has the same.

Reply:
Python 3.6.9 worked for me :)